packages/hub/services/card-service.ts

```diff
@@ -107,7 +109,11 @@ export class CardService implements CardServiceInterface {
   async create(raw: RawCard<Unsaved>): Promise<Card> {
     let compiler = this.builder.compileCardFromRaw(raw);
     let compiledCard = await compiler.compile();
-    let rawCard = await this.realmManager.create(raw);
+    let allFields = fieldsAsList(compiledCard.fields);
+    let { id, realm, adoptsFrom } = raw;
+    let rawCard = await this.realmManager.create(
+      merge({ id, realm, adoptsFrom, data: makeEmptyCardData(allFields) } as RawCard, raw)
```

Comment:
Until we internalize the serializer into the card model, we'll still need to provide the shape of the card's data here since this bypasses the card model when creating a card